docs/buttons/index.md

```diff
@@ -11,4 +13,14 @@ Buttons are used to perform actions, with different visuals for different needs.
 
 <button-example />
 
-<button-tabs />
+<tabs-content> 
+  <template v-slot:react>
+   <react />
+  </template>
+  <template v-slot:vue>
+    <vue />
+  </template>
+  <template v-slot:elements>
+    <elements />
+  </template>
+</tabs-content>
```

Comment:
There should be a way to have that in a component itself but let's improve next week